src/Hangfire.Tags/GlobalConfigurationExtensions.cs

```diff
@@ -24,7 +24,10 @@ public static IGlobalConfiguration UseTags(this IGlobalConfiguration configurati
             if (configuration == null)
                 throw new ArgumentNullException(nameof(configuration));
 
-            TagsOptions.Options = options ?? new TagsOptions { Storage = JobStorage.Current as ITagsServiceStorage };
+            options = options ?? new TagsOptions { Storage = JobStorage.Current as ITagsServiceStorage };
+            options.RetryPolicy = options.RetryPolicy ?? new NoOpTagsStorageTransactionRetryPolicy();
```

Comment:
I actually tested this case with `SqlServer` storage as well. `SqlServer` works just fine without any retry-policies, so we have a `no op` for other storages.

I wasn't able to test `MySqlStorage` because it didn't work for me at all, unfortunately. As for `Redis` storage, I don't have access to `Hangfire.Pro`, so couldn't test it.

Reply:
They all work fine, although we may have to implement an RetryPolicy for `SqlServer` at some point to properly support Azure SQL.